.gitignore

```diff
@@ -0,0 +1 @@
+*.gif
```

Comment:
Please namespace this, otherwise we can never submit a gif source file to this repo.